cpu/cc2538/dev/cc2538-ccm-star.c

```diff
@@ -75,49 +75,18 @@ restore_crypto(uint8_t enabled)
 static void
 set_key(const uint8_t *key)
 {
-  cc2538_aes_128_driver.set_key(key);
+  ccm_star_driver.set_key(key);
```

Comment:
@kkrentz I had a range issue because of the lack of antenna as suggested by @alignan, but I do not have antennae at hand right now. Yet, the basic examples like `examples/ipv6/simple-udp-rpl` work fine without these antennae, so there had to be something else.

I've found that `secmac_driver.init()` calls `configure_radio()`, which calls
`cc2538_rf_driver.set_value(RADIO_PARAM_CCA_THRESHOLD, CCA_THRESHOLD)`,
which does this:
```
REG(RFCORE_XREG_CCACTRL0) = ((int8_t) value) + RSSI_OFFSET;
```

This is wrong. It should be:
```
REG(RFCORE_XREG_CCACTRL0) = (uint8_t)(value + RSSI_OFFSET);
```

`RSSI_OFFSET` is defined as `73`, as per the CC2538 UG. `CCA_THRESHOLD` is defined as `(-50)`. Hence, the `potr` examples set the CCA threshold to -50 dBm (0x17), instead of the -81 dBm (0xF8) recommended by the UG. I've tested with `CCA_THRESHOLD` set to `(-81)`, and the RF range is back to normal. However, there are still `NOACK`s for about 1/3 of packets. These `NOACK`s correspond to `no SHR detected` on the receiver side.

I've also tried to duplicate the reboot issue on the receiver with the original `cc2538-ccm-star.c`, but to no avail. It never reboots, even after a few tens of thousands of `OK`s.